deploy/with-rbac.yaml

```diff
@@ -26,6 +26,7 @@ spec:
             - --configmap=$(POD_NAMESPACE)/nginx-configuration
             - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
             - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
+            - --publish-service=$(POD_NAMESPACE)/ingress-nginx
```

Comment:
Just curious, whats this for?

Reply:
The flag `--publish-service` is used to extract information about the Service fronting the ingress controllers. The controller will set the endpoint records on the ingress objects to reflect those on the service. If you don't use this flag the controller uses the IP address of the node